src/test/java/mesosphere/marathon/core/launchqueue/impl/ReviveOffersStreamLogicTest.scala

```diff
@@ -112,6 +112,24 @@ class ReviveOffersStreamLogicTest extends AkkaUnitTest with Inside {
       logic.handleTick() shouldBe Nil
 
     }
+
+    "send a repeat revive once after the second tick" in {
+      val logic = new ReviveOffersStreamLogic.ReviveRepeaterLogic
+
+      logic.processRoleDirective(UpdateFramework(Map("role" -> OffersWanted), Set("role"), Set.empty))
+      logic.handleTick() shouldBe Nil
+
+      // First repeat for update framework
+      logic.handleTick() shouldBe List(IssueRevive(Set("role")))
+
+      // Revive was triggered
+      logic.processRoleDirective(IssueRevive(Set("role")))
+      logic.handleTick() shouldBe Nil
+
+      // Second repeat for newly triggered revive
+      logic.handleTick() shouldBe List(IssueRevive(Set("role")))
+      logic.handleTick() shouldBe Nil
+    }
```

Comment:
@timcharper, why are doing this every other tick? What is the advantage? Would this increase the revive interval?